int/tests/test_connections.py

```diff
@@ -80,6 +81,38 @@ async def test_create_connection(connection, wait_for_message):
     assert received["label"] == msg_invitation["label"]
 
 
+# Current agent structure in the test suite does not support this testing flow.
+# We'll adjust the agent structure to support this test, but due to competing priorities that will have to wait
+@pytest.mark.skip
```

Comment:
I believe we can include a reason in the `skip` marker; we should add at least some of the text from your comment to this reason.